content/news/playtest-20170303.md

```diff
@@ -0,0 +1,62 @@
+![High resolution font rendering](/images/news/20170303-gapgen.png)
+
+Red Alert Gap Generators now actively reduce enemy unit sight.
+</div>
+
+<div style="text-align:center" markdown="1">
+![High resolution font rendering](/images/news/20170303-hidpi.png)
+
+Players with high resolution laptop or 4k displays will appreciate crisper font rendering.
+</div>
+
+<div class="about-todo-divider"></div>
+
+The biggest changes in this playtest are behind the scenes; technical improvements that are not visible to players, but important for our development and modding community:
+
+* A new [trait conditions](https://github.com/OpenRA/OpenRA/wiki/Trait-Conditions) system provides much better control over trait interactions.
+* Actor vision has been overhauled, adding support for features like the RA active gap generators, D2K actors revealing themselves when attacking, and cliffs blocking vision in heightmap-enabled mods.
+* Simplified weapon definitions using inherited yaml templates.
+* Some initial steps towards removing code patterns that are not compatible with save game support.
+
+OpenRA grew out of the C&C modding community, and one of our major goals is to provide a flexible game engine that the community can use for their own projects.  Unfortunately, this goal has been hurt by OpenRA's single-engine approach, meaning that every new release would unavoidably break community mods.
+
+After many discussions, we have developed a new mod support strategy that we will implement over the next two OpenRA releases.  This playtest (and the subsequent final release) starts by adding support for parallel OpenRA installs, and have started developing a [mod template](https://github.com/OpenRA/OpenRAModTemplate) that includes instructions and scripts to simplify mod development and packaging.  Mods built using this template will be self contained games that function independently from our official mod installs.
+
+The in-game multiplayer server list will now list games for *all* the mods that a player has installed (even across multiple installations), and will allow players to directly switch to another mod when joining one.  This applies across versions as well as mods, meaning that future OpenRA releases will be able to coexist, and switching between release and playtest servers will be just as easy as switching between RA and TD servers today.
+
+<div style="text-align:center" markdown="1">
+![New MP server list switcher](/images/news/20170303-mpmodswitch.png)
+
+The multiplayer server list is now smarter about multiple OpenRA installations.
+</div>
+
+The release after next (aiming for mid/late 2017) will complete the mod-support transition by removing the in-game mod chooser and support for manually installed mod packages.  All mods (both official and community) will be launched directly from the player's operating system.  We are delaying our previously announced project logo changes until this next release to minimize disruption.
```

Comment:
I was very tempted to not mention this... but it would be even worse if we pretended that it didn't happen, because we *are* going to make this change in the future.